CONTRIBUTING.md

```diff
@@ -109,7 +109,13 @@ ever need to write a test for a particular version of Flow, you can put the
 
 #### 5) Run your tests with `flow-typed validate-defs definitions && flow-typed run-tests left-pad`
 
-You may also leave off the argument `left-pad` to run *all* tests (this takes a while). Please note that this test (and the one on Travis-CI) only will be able to run if the name of the repo folder is still "flow-typed".
+```
+cd ../cli && \
+node dist/cli.js validate-defs ../definitions && \
+node dist/cli.js run-tests left-pad
+```
```

Comment:
Thanks for the pull request!

The instructions assume that you have flow-typed installed globally. This is an alternative way but might need some elaboration. This does not work if run from root folder of flow-typed. Could you flesh out a bit enhanced instructions here?

Reply:
Thank you very much for the review

> The instructions assume that you have flow-typed installed globally.

I hope you meant only `5) Run your tests ...` related instructions.
Because I wouldn't say that whole `CONTRIBUTING.md` assumes flow-typed installed globally.

Anyway please take a look I've just updated the instructions.